incubator/chartmuseum/Chart.yaml

```diff
@@ -8,3 +8,5 @@ icon: https://raw.githubusercontent.com/chartmuseum/chartmuseum/master/logo.png
 maintainers:
 - name: ChartMuseum
   email: chartmuseum@gmail.com
+- name: Cloud Posse
+  email: hello@cloudposse.com
```

Comment:
Any reason why many files changed permissions from 644 to 755?

Reply:
Nope. 
Fixed